test-runner/src/main/resources/dcos-application.conf

```diff
@@ -0,0 +1,4 @@
+spark.role = "*"
+spark.attributes = "spark:only"
+spark.roleCpus = "2"
+test.timeout = 5 minutes
```

Comment:
Please add the AWS keys with some dummy values, or commented out.

Should this file be a `.template`, as the other one?